src/utils/url.js

```diff
@@ -195,6 +197,47 @@ export function addQueryParam(url, param, value) {
   return url + fragment;
 }
 
+/**
+ * @param {!../proto/api_messages.Message} data
+ * @return {*}
+ */
+function prepForSerialization(data) {
+  if (data === null || data === undefined) {
+    return undefined;
+  }
+  let arr = null;
+  switch (typeof data) {
+    case 'boolean':
+      return data ? 1 : 0;
```

Comment:
`true` and `false` are valid values in JSON, right? If so, why convert to numbers here?

Reply:
That's what the original clearcut logging JavaScript was doing and 1 is shorter than 'true' or 'false' making it a more compact representation.

I'm open to not making that flip though, it does make the testing harder.

Reply:
Removed this